test/relations.test.js

```diff
@@ -267,6 +267,58 @@ describe('relations', function() {
         }
       });
 
+      it('should fetch all scoped instances with getAsync with callback and options', function(done) {
+        Book.create(function(err, book) {
+          book.chapters.create({name: 'a'}, function() {
+            book.chapters.create({name: 'z'}, function() {
+              book.chapters.create({name: 'c'}, function() {
```

Comment:
This code is ignoring any errors reported by `create` functions. Please fix that.

```js
Book.create(function(err, book) {
  if (err) return done(err);
  // ...
});
```